src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -96,8 +96,8 @@ def handle_exception(ex):  # pylint: disable=too-many-locals, too-many-statement
             az_error = azclierror.ValidationError(error_msg)
 
         elif isinstance(ex, CLIError):
-            # TODO: Fine-grained analysis here for Unknown error
-            az_error = azclierror.UnknownError(error_msg)
+            # TODO: Fine-grained analysis here
+            az_error = azclierror.UnclassifiedUserFault(error_msg)
```

Comment:
Why CLIError here is UserFault?

Reply:
Ideally, `CLIError` will not show up here any more. Temporarily count the CLIError here as UserFaults for better Telemetry analysis.